packages/manager/modules/billing/src/payment/method/add/views/legacyBillingAddress/index.html

```diff
@@ -1,103 +1,116 @@
 <div class="billing-payment-method-add-legacy-bank-account-owner">
-    <!-- Owner Name -->
-    <oui-field
-        data-label="{{ :: 'billing_payment_method_add_bank_account_owner_name' | translate }}"
-    >
-        <input
-            type="text"
-            class="oui-input"
-            id="ownerName"
-            name="ownerName"
-            data-ng-model="$ctrl.model.billingAddress.ownerName"
-            data-required
-            data-ng-minlength="5"
-            data-ng-maxlength="255"
-        />
-    </oui-field>
-    <!-- End of Owner Name -->
-
-    <div data-ng-if="$ctrl.currentUser.billingCountry === 'FR'">
-        <!-- Address Number -->
-        <oui-field
-            data-label="{{ :: 'billing_payment_method_add_bank_account_address_number' | translate }}"
-        >
-            <input
-                type="text"
-                class="oui-input"
-                id="addressNumber"
-                name="addressNumber"
-                data-ng-model="$ctrl.model.billingAddress.addressNumber"
-                data-ng-maxlength="255"
-            />
-        </oui-field>
-        <!-- End of Address Number -->
-
-        <!-- Address Street -->
+    <div class="col-md-6">
```

Comment:
https://github.com/twbs/bootlint/wiki/E013, maybe you should move the `row` class here as well?